### ISSUE TITLE
'Tensor' object has no attribute 'append'

### ISSUE THREAD

**Issue by pengweimin** [tensorflow/models#7539]
*************************************
convnets_list1.append(convnet）
******************************************************
def build_convpool_lstm(input_image, nb_classes, image_size=32, n_colors=1, 
    n_timewin=14, dropout_rate=0.5, num_units=128, batch_size=32, name='CNN_LSTM', train=True, reuse=False):

    with tf.name_scope(name):
        with tf.name_scope('Parallel_CNNs'):
            convnets_list1=[]
            convnets_list2=[]
            convnets=[]
            for i in range(n_timewin):
                for j in range(n_depth):
                    if i==0 and j==0:
                        convnet = build_cnn(input_image[:,i,:,:,j,:],image_size=image_size,n_colors=n_colors, reuse=reuse)
                        
                    else:
                        convnet = build_cnn(input_image[:,i,:,:,j,:],image_size=image_size,n_colors=n_colors, reuse=True)

                    convnets_list1.append(convnet)#18,None,4*4*128
                convnets_list1=tf.stack(convnets_list1)
                convnets_list1=tf.reshape(convnets_list1,shape=[n_depth,-1,4*4*128],name='Reshape_2Dcnn')
                for i in range(convnets_list1.shape[2]):
                    if i==0:
                        convnet=build_1dcnn(convnets_list1[:,:,i],reuse=reuse)
                    else:
                        convnet=build_1dcnn(convnets_list1[:,:,i],reuse=True)
                    convnets_list2.append(convnet)#2048,None,3,32
                convnets_list2= tf.stack(convnets_list2)#14,2048,None,3,32
            convnets.append(convnets_list2)
            convnets=tf.stack(convnets)
            convnets = tf.transpose(convnets, [0,2,1,3,4]) # 调换轴 shape: 14,None,2048,3,32
            convnets = tf.transpose(convnets, [1,0,2,3,4])#None,14,2048,3,32
        with tf.variable_scope('LSTM_layer'):
            # (nSamples, n_timewin, 2, 4, 4, 128) ==>  (nSamples, n_timewin, 5*4*4*256)
            convnets = tf.reshape(convnets, shape=[-1, n_timewin, 2048*3*32], name='Reshape_for_lstm')
            #lstm cell inputs:[batchs, time_steps, 2*4*4*128]
            with tf.variable_scope('LSTM_Cell'):
                lstm_cell = tf.contrib.rnn.BasicLSTMCell(num_units=num_units, forget_bias=1.0, state_is_tuple=True)
                outputs, final_state = tf.nn.dynamic_rnn(lstm_cell, convnets, dtype=tf.float32, time_major=False)
                # outputs.shape is (batch_size, time_steps, num_units)
                outputs = tf.transpose(outputs, [1,0,2])        # (time_steps, batch_size, num_units)
                outputs = outputs[-1]

        with tf.variable_scope('Output_layers'):
            h_fc1_drop1 = tf.layers.dropout(outputs, rate=dropout_rate, training=train, name='dropout_1')
            h_fc1 = tf.layers.dense(h_fc1_drop1, 256, activation=tf.nn.relu, name='fc_relu_256')
            h_fc1_drop2 = tf.layers.dropout(h_fc1, rate=dropout_rate, training=train, name='dropout_2')
            prediction = tf.layers.dense(h_fc1_drop2, nb_classes, name='fc_softmax')

    return prediction

**Comment by pengweimin** [comment]
convnets_list1=[ ]  ,   why this problem?

**Comment by ravikyram** [comment]
@pengchongjin 

Please, let us know which pretrained model you are using and share related code .Thanks!

**Comment by ravikyram** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
